event.go

```diff
@@ -49,7 +49,7 @@ func CreateEventWithHTTPClient(e Event, client HTTPClient) (*EventResponse, erro
 	req.Header.Set("Content-Type", "application/json")
 	resp, err := client.Do(req)
 	if err != nil {
-		return &EventResponse{HttpStatus: resp.StatusCode}, err
+		return  nil, fmt.Errorf("failed to action request: %w", err)
```

Comment:
Would you be able to `go fmt` this file? Just noticed the whitespace before `nil` when getting ready to merge.